webapp/fmprod/src/main/java/org/apache/oodt/cas/product/jaxrs/enums/ErrorTypes.java

```diff
@@ -23,7 +23,7 @@
  *
  * @author ngimhana (Nadeeshan Gimhana)
  */
-public enum ErrorType {
+public enum ErrorTypes {
```

Comment:
I think we agreed to use singular names for the enum type name, `ErrorType` in a last PR